Nodejs/Product/Profiling/source.extension.vsixmanifest

```diff
@@ -1,34 +1,25 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Vsix xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" Version="1.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2010">
-  <Identifier Id="B515653F-FB69-4B64-9D3F-F1FCF8421DD0">
-    <Name>Node.js Tools - Profiling</Name>
-    <Author>Microsoft</Author>
-    <Version>1.2</Version>
+﻿<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="B515653F-FB69-4B64-9D3F-F1FCF8421DD0" Version="1.2" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>Node.js Tools - Profiling</DisplayName>
     <Description xml:space="preserve">Provides support for profiling Node.js projects</Description>
-    <MoreInfoUrl>http://go.microsoft.com/fwlink/?LinkId=785971</MoreInfoUrl>
+    <MoreInfo>http://go.microsoft.com/fwlink/?LinkId=785971</MoreInfo>
     <GettingStartedGuide>http://go.microsoft.com/fwlink/?LinkId=785972</GettingStartedGuide>
-    <Locale>1033</Locale>
-    <InstalledByMsi>true</InstalledByMsi>
     <Icon>NodeJS.ico</Icon>
     <PreviewImage>NodeJS_200x.png</PreviewImage>
-    <SupportedProducts>
-      <VisualStudio Version="11.0">
-        <Edition>Ultimate</Edition>
-        <Edition>Premium</Edition>
-        <Edition>Pro</Edition>
-      </VisualStudio>
-    </SupportedProducts>
-    <SupportedFrameworkRuntimeEdition MinVersion="4.0" />
-  </Identifier>
-  <References>
-    <Reference Id="Microsoft.VisualStudio.MPF" MinVersion="10.0">
-      <Name>Visual Studio MPF</Name>
-    </Reference>
-    <Reference Id="FE8A8C3D-328A-476D-99F9-2A24B75F8C7F" MinVersion="1.2" MaxVersion="1.2">
-      <Name>Node.js Tools for Visual Studio</Name>
-    </Reference>
-  </References>
-  <Content>
-    <VsPackage>|%CurrentProject%;PkgdefProjectOutputGroup|</VsPackage>
-  </Content>
-</Vsix>
+  </Metadata>
+  <!-- Version="11.0" specifies the minimum (not the maximum) version, and the
+       Id specifications are minimums; any SKU equal or 'higher' will accept
+       them. -->
+  <Installation>
+    <InstallationTarget Version="[14.0.25123,16.0)" Id="Microsoft.VisualStudio.Community" />
+    <InstallationTarget Version="[14.0.25123,16.0)" Id="Microsoft.VisualStudio.Pro" />
+    <InstallationTarget Version="[14.0.25123,16.0)" Id="Microsoft.VisualStudio.Enterprise" />
+    <InstallationTarget Version="[14.0.25123,16.0)" Id="Microsoft.VisualStudio.VSWinExpress" />
+    <InstallationTarget Version="[14.0.25123,16.0)" Id="Microsoft.VisualStudio.VWDExpress" />
+    <InstallationTarget Version="[14.0.25123,16.0)" Id="Microsoft.VisualStudio.VSWinDesktopExpress" />
```

Comment:
the installation target should include earlier versions as well otherwise it won't install on RTM